Source/WebCore/WebCore.xcodeproj/project.pbxproj

```diff
@@ -2665,7 +2665,7 @@
 		7132445120109DA500AE7FB2 /* WebAnimationUtilities.h in Headers */ = {isa = PBXBuildFile; fileRef = 7132444F20109D9B00AE7FB2 /* WebAnimationUtilities.h */; settings = {ATTRIBUTES = (Private, ); }; };
 		7134496E146941B300720312 /* SVGLengthContext.h in Headers */ = {isa = PBXBuildFile; fileRef = 7134496C146941B300720312 /* SVGLengthContext.h */; settings = {ATTRIBUTES = (Private, ); }; };
 		713516932AE6D01F00718EBE /* ScrollTimeline.h in Headers */ = {isa = PBXBuildFile; fileRef = 713516912AE6D00A00718EBE /* ScrollTimeline.h */; settings = {ATTRIBUTES = (Private, ); }; };
-		713516982AE6D2AE00718EBE /* ScrollAxis.h in Headers */ = {isa = PBXBuildFile; fileRef = 713516962AE6D29600718EBE /* ScrollAxis.h */; };
+		713516982AE6D2AE00718EBE /* ScrollAxis.h in Headers */ = {isa = PBXBuildFile; fileRef = 713516962AE6D29600718EBE /* ScrollAxis.h */; settings = {ATTRIBUTES = (Private, ); }; };
```

Comment:
Is this change on purpose?

Reply:
Yes, this addresses a compilation error (which seems to happen on non-Apple ports, so something I’ll need to fix).